Makefile

```diff
@@ -105,4 +105,4 @@ bench-tx: ## Run transaction benchmarks
 
 .PHONY: install-prover
 install-prover: ## Installs prover
-	cargo install --path bin/tx-prover --locked
+	cargo install --path bin/tx-prover --locked --features testing
```

Comment:
If we keep this, we should probably update the above comment on line 107. But  I'm not sure we want to keep this change in the first place.

Reply:
Maybe we can have two install commands like in the node